packages/presentation/lib/utils/funcs/capitalize.dart

```diff
@@ -0,0 +1,6 @@
+String capitalize(String? s) {
+  if (s == null) {
+    return 'Err';
+  }
+  return s[0].toUpperCase() + s.substring(1).toLowerCase();
+}
```

Comment:
1) Idea is great!!
2) let's move it to domain level (I don't think it is related only to presentation)
3) imho, for a null case we should return null (up to u)
4) make it as a string extension